### PR TITLE
DEV-57: create main chat 

### PR DESCRIPTION
**Pull request by yefimchuk**
## Ticket
Link to ticket:
https://bsalo.atlassian.net/browse/DEV-57?atlOrigin=eyJpIjoiYTA0YmE5NjY2NmM4NDhjNWI2ZGJlN2MxN2I5ODczYTAiLCJwIjoiaiJ9

## Description
*Please provide a small summary of your changes and how they resolved the issues in the ticket:*

- Create main chat component
## Screenshoot

image

## Checklist

- [] Did you test it?
- [] Did you add tests?
- [] Did you add to the storybook?

### CODE REVIEW

File: src/pages/Chat/Chat.scss [hunk 1]
```diff
@@ -0,0 +1,46 @@
+.main
+{
+  display: flex;
```
Comment (Skitus) [inline src/pages/Chat/Chat.scss:3]: pls don`t use flex and other styles we have antd

File: src/pages/Chat/Chat.scss [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 700px;
+
+}
+.menu
+{
+  display: flex;
```
Comment (Skitus) [inline src/pages/Chat/Chat.scss:13]: pls check antd layout and fix it

File: src/pages/Chat/Chat.tsx [hunk 3]
```diff
@@ -1,11 +1,32 @@
-import { Button } from 'antd';
 import React from 'react';
 import './Chat.scss';
```
Comment (Skitus) [inline src/pages/Chat/Chat.tsx:2]: pls push it import to he ends of the imports

File: src/pages/Chat/Chat.tsx [hunk 4]
```diff
@@ -1,11 +1,32 @@
...
+import 'antd/dist/antd.css';
+import { MailOutlined, MessageOutlined, CalendarOutlined, SmileOutlined } from '@ant-design/icons';
+import { Switch } from 'antd';
 
 const Chat = () => (
   <section>
-    <h1>Chat page</h1>
-    <Button> Hello world </Button>
+    <div className="main">
```
Comment (Skitus) [inline src/pages/Chat/Chat.tsx:9]: dont use your own styles and layouts pls use antd

Comment (Skitus) [reply src/pages/Chat/Chat.tsx:9]: fix all this file

File: src/pages/Chat/Chat.tsx (hunk 3 above)
Comment (Skitus) [reply src/pages/Chat/Chat.tsx:2]: import './Chat.scss';

**Review by dev1B — CHANGES_REQUESTED** [review]

**Comment by dev1B** [comment]
Need to complete and reopen. No updates for a long time

**Pull request closed without merging**